engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -86,20 +90,18 @@
     private final EntityManager entityManager;
 
     private final Camera playerCamera;
-    private final Camera shadowMapCamera = new OrthographicCamera(-SHADOW_FRUSTUM_BOUNDS, SHADOW_FRUSTUM_BOUNDS, SHADOW_FRUSTUM_BOUNDS, -SHADOW_FRUSTUM_BOUNDS);
 
     // TODO: Review this? (What are we doing with a component not attached to an entity?)
     private LightComponent mainDirectionalLight = new LightComponent();
     private float timeSmoothedMainLightIntensity;
 
-    private final RenderQueuesHelper renderQueues;
+
```

Comment:
Now that I think about it, I've noticed this happening a number of times. Please go through the code and eliminate lines you left empty. Of course you can leave empty lines that were already there or if you feel that it make sense to leave an additional empty line.